gr-trellis/include/gnuradio/trellis/fsm.h

```diff
@@ -14,6 +14,7 @@
 #include <gnuradio/trellis/api.h>
 #include <iosfwd>
 #include <vector>
+#include <string>
```

Comment:
Can you please clang-format this -- it should move the include further up.

Reply:
done

Reply:
cheers m8